search_service/proxy/elasticsearch.py

```diff
@@ -23,6 +23,15 @@
 
 LOGGING = logging.getLogger(__name__)
 
+# mapping to translate request
+TABLE_MAPPING = {
+    'tag': 'tags',
+    'schema': 'schema_name.raw',
+    'table': 'name.raw',
+    'column': 'column_names.raw',
+    'database': 'database.raw'
+}
+
```

Comment:
This isn't something we considered -- but it would affect the rankings and I assume these are value mappings that you've been using internally and yield good results? If so, I'd be open to adding them in a future PR as part of what I think will be an ongoing community effort to improve search relevancy. 

The only reason I'm not personally keen on adding them now is that I prefer to take an iterative approach given we haven't vetted the behavior this new endpoint in production at Lyft yet, and I'd be curious to do some experiments on our end as well so that we can align on good defaults.

Reply:
Actually, rankings already exist in the current amundsenssearchlibrary in a few methods - fetch_table_search_results_with_field, fetch_table_search_results and fetch_user_search_results methods. The mapping I'm sharing is from our code (and the PR I put up on search) but also is pulled basically directly from what already existed in this library

Reply:
maybe worth put a comment/todo in the mapping to consolidate all the mapping with ranking together ?